lib/EncodeImagesStage.js

```diff
@@ -4,20 +4,32 @@ var Promise = require('bluebird');
 
 Jimp.prototype.getBufferAsync = Promise.promisify(Jimp.prototype.getBuffer);
 
-module.exports = encodeImages;
+module.exports = EncodeImagesStage;
+
+/**
+ * Encodes Jimp images in pipeline extras to buffers.
+ *
+ * The glTF asset must be initialized for the pipeline.
+ */
+function EncodeImagesStage() {
+    this.name = 'encodeImages';
+    this.before = ['quantizeAttributes'];
```

Comment:
I don't think it does. But I think I'll probably draw a dependency graph before moving forward so that the dependencies are established correctly.